sql/moz-fx-data-shared-prod/firefox_ios/retention_clients/view.sql

```diff
@@ -0,0 +1,93 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_ios.retention_clients`
+AS
+WITH clients_last_seen AS (
+  SELECT
+    submission_date,
+    client_id,
+    sample_id,
+    normalized_channel,
+    mozfun.bits28.retention(days_seen_bits, submission_date) AS retention_seen,
+    mozfun.bits28.retention(days_active_bits & days_seen_bits, submission_date) AS retention_active,
+    days_seen_bits,
+    days_active_bits,
+  FROM
+    `moz-fx-data-shared-prod.firefox_ios.baseline_clients_last_seen`
+),
+attribution AS (
+  SELECT
+    client_id,
+    sample_id,
+    channel AS normalized_channel,
+    adjust_ad_group,
+    adjust_campaign,
+    adjust_creative,
+    adjust_network,
+    is_suspicious_device_profile,
+  FROM
+    `moz-fx-data-shared-prod.firefox_ios_derived.firefox_ios_clients_v1`
+)
+SELECT
+  clients_last_seen.submission_date AS submission_date,
+  clients_daily.submission_date AS metric_date,
+  clients_daily.first_seen_date,
+  clients_daily.client_id,
+  clients_daily.sample_id,
+  clients_last_seen.app_name,
+  clients_daily.normalized_channel,
+  clients_daily.country,
+  clients_daily.app_display_version AS app_version,
+  clients_daily.locale,
+  clients_daily.isp,
+  attribution.is_suspicious_device_profile,
+  -- ping sent retention
+  clients_last_seen.retention_seen.day_27.active_on_metric_date AS ping_sent_metric_date,
+  (
+    clients_last_seen.retention_seen.day_27.active_on_metric_date
+    AND clients_last_seen.retention_seen.day_27.active_in_week_3
+  ) AS ping_sent_week_4,
+  -- activity retention
+  clients_last_seen.retention_active.day_27.active_on_metric_date AS active_metric_date,
+  (
+    clients_last_seen.retention_active.day_27.active_on_metric_date
+    AND clients_last_seen.retention_active.day_27.active_in_week_3
+  ) AS retained_week_4,
+  -- new profile retention
+  clients_daily.is_new_profile AS new_profile_metric_date,
+  (
+    clients_daily.is_new_profile
+    AND clients_last_seen.retention_active.day_27.active_in_week_3
+  ) AS retained_week_4_new_profile,
+  (
+    clients_daily.is_new_profile
+    AND BIT_COUNT(clients_last_seen.days_active_bits) > 1
+  ) AS repeat_client,
```

Comment:
We missed one: `repeat_profile`.
It's already reading correctly on the aggregate table and everything looks correct for Android.